### ISSUE TITLE
[Bug] Shift + Opt + Right/Left Arrow override?

### ISSUE THREAD

**Issue by alexa-llibre** [piroor/treestyletab#3575]
Hello, after updating to the newest version of TST and changing no other settings for TST or anywhere on my computer (Mac), whenever I try to use the Shift + Opt + Right/Left Arrow keyboard shortcut to select text, it instead collapses the TST tree the selected tab belongs to. If I press those keys again immediately afterward it will move to the previous or next tab. I believe this is related to the TST new shortcut (from the release notes: "Add new keyboard shortcut slots "Simulate Up/Down/Left/Right Key on Tree" and assign default shortcuts Alt-Shift-Up/Down/Left/Right. They provide both spatial focus move on faviconized tabs and collapse/expand tree like arrow keys on tree items."). 

I need to be able to opt out of the TST keyboard shortcut to keep my shortcuts as is (i.e. Shift Option Right/Left Arrow should select text, not do anything with TST). Thank you for your help!

 * Platform (OS): Mac, Sonoma 14.5
 * Version of Firefox: 126.0.1 
 * Version (or revision) of Tree Style Tab: Version 4.0.17

**Comment by CodingKoopa** [comment]
This seems to have been introduced with 4.0.16 (2024.6.9):

> Add new keyboard shortcut slots "Simulate Up/Down/Left/Right Key on Tree" and assign default shortcuts Alt-Shift-Up/Down/Left/Right. They provide both spatial focus move on faviconized tabs and collapse/expand tree like arrow keys on tree items.

Similar issue to #1995.

**Comment by piroor** [comment]
Sorry I didn't know "Alt" has different function on macOS. I've pushed a change to reassign default shortcuts as "MacCtrl+Shift+Up/Down/Left/Right".

And you can unassign those shortcuts with the addons manager:
![image](https://github.com/piroor/treestyletab/assets/70062/8a061213-4b4d-4ea0-857d-2d91079100a3)

**Comment by mfn** [comment]
Thanks, that also fixed it for me. Was quite confused, because such a keystroke in a text area (like here on github) suddenly jumped to a different tab 💥

**Comment by dimitarvp** [comment]
@piroor This fixed it for me, thanks a lot.

For reference: I landed here from https://connect.mozilla.org/t5/discussions/option-shift-arrow-now-switches-tabs-since-v-127/m-p/59302 and https://bugzilla.mozilla.org/show_bug.cgi?id=1901978

**Comment by alexa-llibre** [comment]
Thank you! Also fixed for me - appreciate your quick response!

**Comment by codeboten** [comment]
Thank you for the details! This will save me many minutes of frustration on a daily basis!

**Comment by isometricmax** [comment]
Got here from the firefox subreddit, and @piroor's solution fixed it. Thanks!

**Comment by gotofritz** [comment]
This was so irritating I almost ditched Firefox for Brave... luckily someone on the Mozilla forum pointed me here.
Please don't do this again 😄

**Comment by ionut-mntn** [comment]
Indeed very irritating. Glad it's solved & found this: https://support.mozilla.org/en-US/questions/1449863 which led me here.


All good now
![image](https://github.com/piroor/treestyletab/assets/43119550/bff3f5ca-5198-4902-b6b3-abb9791eec55)
All good now

Btw you mention you pushed it.. but does that mean it also reached prod? ie: I used Tamper Monkey to install Tree Style Tabs extension and usually whenever I open up firefox - if there is any update, then Tamper Monkey asks me if I want to install it (in case I have not configured that specific script to be auto-updated)... and saw no no update for the Tree Style Tabs today when I opened my browser.
